watertap/property_models/ion_DSPMDE_prop_pack.py

```diff
@@ -440,7 +517,7 @@ def _mass_frac_phase_comp(self):
             self.params.phase_list,
             self.params.component_list,
             initialize=lambda b,p,j : 0.4037 if j == "H2O" else 0.0033, #todo: revisit
-            bounds=(1e-6, 1.001),
+            bounds=(1e-8, 1.001),
```

Comment:
This is more of comment on how we bound variables like this in general: why over-relax the upper bound but under-relax the lower bound?

If we're trying to avoid 0, I think we should settle on a standard epsilon for the codebase. Sometimes these bounds become binding unexpectedly.